2024/09/22 20:11:06 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "post". in C:\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post')
#1 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/blog/index.php?r=post
---
2024/09/22 20:12:15 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "user". in C:\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/blog/index.php?r=user
---
2024/09/23 09:13:18 [error] [exception.CException] CException: Property "CDbCriteria.0" is not defined. in C:\yii\framework\base\CComponent.php:172
Stack trace:
#0 C:\yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('0', 'user1')
#1 C:\yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 C:\yii\framework\db\ar\CActiveRecord.php(1463): CDbCommandBuilder->createCriteria(Array, Array)
#3 C:\Apache24\htdocs\blog\protected\components\UserIdentity.php(12): CActiveRecord->find(Array)
#4 C:\Apache24\htdocs\blog\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#5 C:\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#8 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/blog/index.php?r=site/login
HTTP_REFERER=http://localhost/blog/index.php?r=site/login
---
2024/09/23 09:13:46 [error] [exception.CException] CException: Property "CDbCriteria.0" is not defined. in C:\yii\framework\base\CComponent.php:172
Stack trace:
#0 C:\yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('0', 'user1')
#1 C:\yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 C:\yii\framework\db\ar\CActiveRecord.php(1463): CDbCommandBuilder->createCriteria(Array, Array)
#3 C:\Apache24\htdocs\blog\protected\components\UserIdentity.php(12): CActiveRecord->find(Array)
#4 C:\Apache24\htdocs\blog\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#5 C:\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#8 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/blog/index.php?r=site/login
HTTP_REFERER=http://localhost/blog/index.php?r=site/login
---
2024/09/23 09:14:08 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'user1' in 'where clause'. The SQL statement executed was: SELECT * FROM `tbl_user` `t` WHERE user1 LIMIT 1.
in C:\Apache24\htdocs\blog\protected\components\UserIdentity.php (12)
in C:\Apache24\htdocs\blog\protected\models\LoginForm.php (54)
in C:\Apache24\htdocs\blog\protected\controllers\SiteController.php (94)
2024/09/23 09:14:08 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'user1' in 'where clause'. The SQL statement executed was: SELECT * FROM `tbl_user` `t` WHERE user1 LIMIT 1 in C:\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\yii\framework\db\CDbCommand.php(415): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\yii\framework\db\ar\CActiveRecord.php(1359): CDbCommand->queryRow()
#2 C:\yii\framework\db\ar\CActiveRecord.php(1464): CActiveRecord->query(Object(CDbCriteria))
#3 C:\Apache24\htdocs\blog\protected\components\UserIdentity.php(12): CActiveRecord->find('user1')
#4 C:\Apache24\htdocs\blog\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#5 C:\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#8 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/blog/index.php?r=site/login
HTTP_REFERER=http://localhost/blog/index.php?r=site/login
---
2024/09/23 09:50:45 [error] [exception.Error] Error: Undefined constant "ERROR_NONE" in C:\Apache24\htdocs\blog\protected\components\UserIdentity.php:22
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#1 C:\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#2 C:\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#3 C:\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#4 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(94): CModel->validate()
#5 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#6 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\yii\framework\web\CWebApplication.php(282): CController->run('login')
#10 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#11 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/blog/index.php?r=site/login
HTTP_REFERER=http://localhost/blog/index.php?r=site/login
---
2024/09/23 11:35:22 [error] [php] Undefined variable $username (C:\Apache24\htdocs\blog\protected\models\LoginForm.php:51)
Stack trace:
#0 C:\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#1 C:\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#2 C:\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#3 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(94): LoginForm->validate()
#4 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\yii\framework\web\CController.php(286): SiteController->runAction()
#7 C:\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 C:\yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\Apache24\htdocs\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/blog/index.php?r=site/login
in C:\Apache24\htdocs\blog\protected\models\LoginForm.php (51)
in C:\Apache24\htdocs\blog\protected\controllers\SiteController.php (94)
in C:\Apache24\htdocs\blog\index.php (13)
2024/09/23 11:39:12 [error] [php] Undefined variable $identity (C:\Apache24\htdocs\blog\protected\models\LoginForm.php:53)
Stack trace:
#0 C:\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#1 C:\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#2 C:\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#3 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(94): LoginForm->validate()
#4 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\yii\framework\web\CController.php(286): SiteController->runAction()
#7 C:\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 C:\yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\Apache24\htdocs\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/blog/index.php?r=site/login
in C:\Apache24\htdocs\blog\protected\models\LoginForm.php (53)
in C:\Apache24\htdocs\blog\protected\controllers\SiteController.php (94)
in C:\Apache24\htdocs\blog\index.php (13)
2024/09/23 11:39:42 [error] [php] include(UserIdenty.php): Failed to open stream: No such file or directory (C:\yii\framework\YiiBase.php:463)
Stack trace:
#0 C:\yii\framework\YiiBase.php(463): include()
#1 C:\Apache24\htdocs\blog\protected\models\LoginForm.php(58): autoload()
#2 C:\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\Apache24\htdocs\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/blog/index.php?r=site/login
in C:\Apache24\htdocs\blog\protected\models\LoginForm.php (58)
in C:\Apache24\htdocs\blog\protected\controllers\SiteController.php (94)
in C:\Apache24\htdocs\blog\index.php (13)
2024/09/23 12:05:31 [error] [php] Undefined variable $identity (C:\Apache24\htdocs\blog\protected\models\LoginForm.php:56)
Stack trace:
#0 C:\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#1 C:\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#2 C:\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#3 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(94): LoginForm->validate()
#4 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\yii\framework\web\CController.php(286): SiteController->runAction()
#7 C:\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 C:\yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\Apache24\htdocs\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/blog/index.php?r=site/login
in C:\Apache24\htdocs\blog\protected\models\LoginForm.php (56)
in C:\Apache24\htdocs\blog\protected\controllers\SiteController.php (94)
in C:\Apache24\htdocs\blog\index.php (13)
2024/09/23 12:06:13 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=site/login
---
2024/09/23 12:23:24 [error] [exception.ParseError] ParseError: syntax error, unexpected token "}", expecting ";" in C:\Apache24\htdocs\blog\protected\models\Post.php:117
Stack trace:
#0 C:\yii\framework\db\ar\CActiveRecord.php(397): YiiBase::autoload('Post')
#1 C:\yii\framework\web\CActiveDataProvider.php(165): CActiveRecord::model('Post')
#2 C:\yii\framework\web\CActiveDataProvider.php(87): CActiveDataProvider->getModel('Post')
#3 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(125): CActiveDataProvider->__construct('Post')
#4 C:\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#5 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#7 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#11 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#12 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\yii\framework\web\CWebApplication.php(282): CController->run('')
#14 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post')
#15 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/blog/index.php?r=post
---
2024/09/23 12:24:22 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=post/index
---
2024/09/23 13:19:18 [error] [exception.CException] CException: Property "CWebUser.username" is not defined. in C:\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\yii\framework\web\auth\CWebUser.php(146): CComponent->__get('username')
#1 C:\Apache24\htdocs\blog\protected\components\UserMenu.php(8): CWebUser->__get('username')
#2 C:\yii\framework\web\CBaseController.php(147): UserMenu->init()
#3 C:\yii\framework\web\CBaseController.php(181): CBaseController->createWidget('UserMenu', Array)
#4 C:\Apache24\htdocs\blog\protected\views\layouts\column1.php(7): CBaseController->widget('UserMenu')
#5 C:\yii\framework\web\CBaseController.php(126): require('C:\\Apache24\\htd...')
#6 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\Apache24\\htd...', Array, true)
#7 C:\yii\framework\web\CController.php(787): CBaseController->renderFile('C:\\Apache24\\htd...', Array, true)
#8 C:\yii\framework\web\actions\CViewAction.php(139): CController->render('pages/about')
#9 C:\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#10 C:\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#11 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#12 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#13 C:\yii\framework\web\CWebApplication.php(282): CController->run('page')
#14 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#15 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/blog/index.php?r=site/page&view=about
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:20:38 [error] [exception.ParseError] ParseError: syntax error, unexpected variable "$this" in C:\Apache24\htdocs\blog\protected\views\layouts\column1.php:10
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\Apache24\\htd...', Array, true)
#1 C:\yii\framework\web\CController.php(787): CBaseController->renderFile('C:\\Apache24\\htd...', Array, true)
#2 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(32): CController->render('index')
#3 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=site/index
HTTP_REFERER=http://localhost/blog/index.php?r=site/contact
---
2024/09/23 13:21:33 [error] [exception.CException] CException: SiteController has an extra endWidget(CContentDecorator) call in its view. in C:\yii\framework\web\CBaseController.php:219
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(309): CBaseController->endWidget('CContentDecorat...')
#1 C:\Apache24\htdocs\blog\protected\views\layouts\column1.php(15): CBaseController->endContent()
#2 C:\yii\framework\web\CBaseController.php(126): require('C:\\Apache24\\htd...')
#3 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\Apache24\\htd...', Array, true)
#4 C:\yii\framework\web\CController.php(787): CBaseController->renderFile('C:\\Apache24\\htd...', Array, true)
#5 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(32): CController->render('index')
#6 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#7 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\yii\framework\web\CWebApplication.php(282): CController->run('index')
#11 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index')
#12 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/blog/index.php?r=site/index
HTTP_REFERER=http://localhost/blog/index.php?r=site/contact
---
2024/09/23 13:22:22 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:22:31 [error] [exception.ParseError] ParseError: syntax error, unexpected variable "$this" in C:\Apache24\htdocs\blog\protected\views\layouts\column2.php:12
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\Apache24\\htd...', Array, true)
#1 C:\yii\framework\web\CController.php(787): CBaseController->renderFile('C:\\Apache24\\htd...', Array, true)
#2 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(77): CController->render('create', Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/create')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/create
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:22:57 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=post/create
---
2024/09/23 13:25:30 [error] [php] Undefined variable $dataProvider (C:\Apache24\htdocs\blog\protected\views\site\index.php:12)
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(126): require()
#1 C:\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\yii\framework\web\CController.php(872): SiteController->renderFile()
#3 C:\yii\framework\web\CController.php(785): SiteController->renderPartial()
#4 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(32): SiteController->render()
#5 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\yii\framework\web\CController.php(286): SiteController->runAction()
#8 C:\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\yii\framework\web\CWebApplication.php(282): SiteController->run()
#10 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\Apache24\htdocs\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/blog/index.php
in C:\Apache24\htdocs\blog\protected\views\site\index.php (12)
in C:\Apache24\htdocs\blog\protected\controllers\SiteController.php (32)
in C:\Apache24\htdocs\blog\index.php (13)
2024/09/23 13:26:08 [error] [php] Undefined variable $dataProvider (C:\Apache24\htdocs\blog\protected\views\site\index.php:14)
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(126): require()
#1 C:\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\yii\framework\web\CController.php(872): SiteController->renderFile()
#3 C:\yii\framework\web\CController.php(785): SiteController->renderPartial()
#4 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(32): SiteController->render()
#5 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\yii\framework\web\CController.php(286): SiteController->runAction()
#8 C:\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\yii\framework\web\CWebApplication.php(282): SiteController->run()
#10 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\Apache24\htdocs\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/blog/index.php
in C:\Apache24\htdocs\blog\protected\views\site\index.php (14)
in C:\Apache24\htdocs\blog\protected\controllers\SiteController.php (32)
in C:\Apache24\htdocs\blog\index.php (13)
2024/09/23 13:28:48 [error] [exception.ValueError] ValueError: Path cannot be empty in C:\yii\framework\web\CBaseController.php:130
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal(false, Array, false)
#1 C:\yii\framework\zii\widgets\CListView.php(291): CBaseController->renderFile(false, Array)
#2 C:\yii\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#3 [internal function]: CBaseListView->renderSection(Array)
#4 C:\yii\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback('/{(\\w+)}/', Array, '{summary}\n{sort...')
#5 C:\yii\framework\zii\widgets\CBaseListView.php(149): CBaseListView->renderContent()
#6 C:\yii\framework\web\CBaseController.php(182): CBaseListView->run()
#7 C:\Apache24\htdocs\blog\protected\views\site\index.php(11): CBaseController->widget('zii.widgets.CLi...', Array)
#8 C:\yii\framework\web\CBaseController.php(126): require('C:\\Apache24\\htd...')
#9 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\Apache24\\htd...', Array, true)
#10 C:\yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\Apache24\\htd...', Array, true)
#11 C:\yii\framework\web\CController.php(785): CController->renderPartial('index', Array, true)
#12 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(31): CController->render('index', Array)
#13 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\yii\framework\web\CWebApplication.php(282): CController->run('')
#18 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#19 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/blog/index.php
---
2024/09/23 13:30:15 [error] [exception.ValueError] ValueError: Path cannot be empty in C:\yii\framework\web\CBaseController.php:130
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal(false, Array, false)
#1 C:\yii\framework\zii\widgets\CListView.php(291): CBaseController->renderFile(false, Array)
#2 C:\yii\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#3 [internal function]: CBaseListView->renderSection(Array)
#4 C:\yii\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback('/{(\\w+)}/', Array, '{summary}\n{sort...')
#5 C:\yii\framework\zii\widgets\CBaseListView.php(149): CBaseListView->renderContent()
#6 C:\yii\framework\web\CBaseController.php(182): CBaseListView->run()
#7 C:\Apache24\htdocs\blog\protected\views\site\index.php(11): CBaseController->widget('zii.widgets.CLi...', Array)
#8 C:\yii\framework\web\CBaseController.php(126): require('C:\\Apache24\\htd...')
#9 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\Apache24\\htd...', Array, true)
#10 C:\yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\Apache24\\htd...', Array, true)
#11 C:\yii\framework\web\CController.php(785): CController->renderPartial('index', Array, true)
#12 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(31): CController->render('index', Array)
#13 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\yii\framework\web\CWebApplication.php(282): CController->run('')
#18 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#19 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/blog/index.php
---
2024/09/23 13:30:48 [error] [exception.CException] CException: The property "itemView" cannot be empty. in C:\yii\framework\zii\widgets\CListView.php:210
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(147): CListView->init()
#1 C:\yii\framework\web\CBaseController.php(181): CBaseController->createWidget('zii.widgets.CLi...', Array)
#2 C:\Apache24\htdocs\blog\protected\views\site\index.php(11): CBaseController->widget('zii.widgets.CLi...', Array)
#3 C:\yii\framework\web\CBaseController.php(126): require('C:\\Apache24\\htd...')
#4 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\Apache24\\htd...', Array, true)
#5 C:\yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\Apache24\\htd...', Array, true)
#6 C:\yii\framework\web\CController.php(785): CController->renderPartial('index', Array, true)
#7 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(31): CController->render('index', Array)
#8 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#9 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php
---
2024/09/23 13:33:09 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "view". in C:\yii\framework\web\CController.php:486
Stack trace:
#0 C:\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/view')
#3 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/blog/index.php?r=site/view&id=1
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:33:53 [error] [exception.CException] CException: The property "itemView" cannot be empty. in C:\yii\framework\zii\widgets\CListView.php:210
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(147): CListView->init()
#1 C:\yii\framework\web\CBaseController.php(181): CBaseController->createWidget('zii.widgets.CLi...', Array)
#2 C:\Apache24\htdocs\blog\protected\views\post\index.php(17): CBaseController->widget('zii.widgets.CLi...', Array)
#3 C:\yii\framework\web\CBaseController.php(126): require('C:\\Apache24\\htd...')
#4 C:\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\Apache24\\htd...', Array, true)
#5 C:\yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\Apache24\\htd...', Array, true)
#6 C:\yii\framework\web\CController.php(785): CController->renderPartial('index', Array, true)
#7 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(126): CController->render('index', Array)
#8 C:\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#9 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#13 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#15 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#16 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\yii\framework\web\CWebApplication.php(282): CController->run('index')
#18 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#19 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/blog/index.php?r=post/index
HTTP_REFERER=http://localhost/blog/index.php?r=post/create
---
2024/09/23 13:34:12 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "view". in C:\yii\framework\web\CController.php:486
Stack trace:
#0 C:\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/view')
#3 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/blog/index.php?r=site/view&id=1
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:34:38 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "view". in C:\yii\framework\web\CController.php:486
Stack trace:
#0 C:\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/view')
#3 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/blog/index.php?r=site/view&id=1
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:34:39 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "view". in C:\yii\framework\web\CController.php:486
Stack trace:
#0 C:\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/view')
#3 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/blog/index.php?r=site/view&id=1
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:34:39 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "view". in C:\yii\framework\web\CController.php:486
Stack trace:
#0 C:\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/view')
#3 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/blog/index.php?r=site/view&id=1
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:34:39 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "view". in C:\yii\framework\web\CController.php:486
Stack trace:
#0 C:\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/view')
#3 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/blog/index.php?r=site/view&id=1
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:36:21 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "view". in C:\yii\framework\web\CController.php:486
Stack trace:
#0 C:\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/view')
#3 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/blog/index.php?r=site/view&id=1
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:36:46 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "view". in C:\yii\framework\web\CController.php:486
Stack trace:
#0 C:\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/view')
#3 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/blog/index.php?r=site/view&id=1
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:36:46 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "view". in C:\yii\framework\web\CController.php:486
Stack trace:
#0 C:\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/view')
#3 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/blog/index.php?r=site/view&id=1
HTTP_REFERER=http://localhost/blog/index.php
---
2024/09/23 13:41:30 [error] [php] Undefined variable $dataProvider (C:\Apache24\htdocs\blog\protected\views\site\index.php:14)
Stack trace:
#0 C:\yii\framework\web\CBaseController.php(126): require()
#1 C:\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\yii\framework\web\CController.php(872): SiteController->renderFile()
#3 C:\yii\framework\web\CController.php(785): SiteController->renderPartial()
#4 C:\Apache24\htdocs\blog\protected\controllers\SiteController.php(34): SiteController->render()
#5 C:\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\yii\framework\web\CController.php(286): SiteController->runAction()
#8 C:\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\yii\framework\web\CWebApplication.php(282): SiteController->run()
#10 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\Apache24\htdocs\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/blog/index.php
in C:\Apache24\htdocs\blog\protected\views\site\index.php (14)
in C:\Apache24\htdocs\blog\protected\controllers\SiteController.php (34)
in C:\Apache24\htdocs\blog\index.php (13)
2024/09/23 13:47:23 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=post/index
---
2024/09/23 15:43:01 [error] [php] Undefined variable $model (C:\Apache24\htdocs\blog\protected\controllers\PostController.php:85)
Stack trace:
#0 unknown(0): PostController->actionUpdate()
#1 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs()
#2 C:\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\yii\framework\web\filters\CFilterChain.php(134): PostController->runAction()
#5 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(1148): CAccessControlFilter->filter()
#7 C:\yii\framework\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#8 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\yii\framework\web\CController.php(265): PostController->runActionWithFilters()
#11 C:\yii\framework\web\CWebApplication.php(282): PostController->run()
#12 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\Apache24\htdocs\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/blog/index.php?r=post/update&id=1
in C:\Apache24\htdocs\blog\protected\controllers\PostController.php (85)
in C:\Apache24\htdocs\blog\index.php (13)
2024/09/23 15:45:54 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/delete')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/delete&id=4
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=4
---
2024/09/23 15:46:08 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/delete')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/delete&id=1
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=1
---
2024/09/23 15:46:35 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=1
---
2024/09/23 15:46:39 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:164
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('1')
#1 [internal function]: PostController->actionView('1')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=1
HTTP_REFERER=http://localhost/blog/index.php?r=post/index
---
2024/09/23 15:46:47 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=2
---
2024/09/23 15:48:07 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=2
---
2024/09/23 15:48:10 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:164
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('2')
#1 [internal function]: PostController->actionView('2')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=2
HTTP_REFERER=http://localhost/blog/index.php?r=post/index
---
2024/09/23 15:48:28 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=3
---
2024/09/23 15:48:34 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:164
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('3')
#1 [internal function]: PostController->actionView('3')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=3
HTTP_REFERER=http://localhost/blog/index.php?r=post/index
---
2024/09/23 15:48:46 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=10
---
2024/09/23 15:48:50 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:164
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('10')
#1 [internal function]: PostController->actionView('10')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=10
HTTP_REFERER=http://localhost/blog/index.php?r=post/index
---
2024/09/23 15:48:53 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\yii\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/blog/index.php?r=post/admin
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=10
---
2024/09/23 15:49:07 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:164
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('10')
#1 [internal function]: PostController->actionView('10')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=10
HTTP_REFERER=http://localhost/blog/index.php?r=post/index
---
2024/09/23 15:52:26 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:161
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('11')
#1 [internal function]: PostController->actionView('11')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=11
HTTP_REFERER=http://localhost/blog/index.php?r=post/index
---
2024/09/23 15:53:35 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\yii\framework\web\CController.php:337
Stack trace:
#0 C:\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#10 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/blog/index.php?r=post/view
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=12
---
2024/09/23 15:53:42 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:161
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('12')
#1 [internal function]: PostController->actionView('12')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=12
HTTP_REFERER=http://localhost/blog/index.php?r=post/update&id=12
---
2024/09/23 15:53:43 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:161
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(85): PostController->loadModel('12')
#1 [internal function]: PostController->actionUpdate('12')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('update')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/update')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/update&id=12
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=12
---
2024/09/23 15:53:44 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:161
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('12')
#1 [internal function]: PostController->actionView('12')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=12
HTTP_REFERER=http://localhost/blog/index.php?r=post/create
---
2024/09/23 15:54:28 [error] [exception.ArgumentCountError] ArgumentCountError: Too few arguments to function CController::redirect(), 0 passed in C:\Apache24\htdocs\blog\protected\controllers\PostController.php on line 115 and at least 1 expected in C:\yii\framework\web\CController.php:1026
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(115): CController->redirect()
#1 [internal function]: PostController->actionDelete('13')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/delete')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/delete&id=13
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=13
---
2024/09/23 15:56:49 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:161
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('14')
#1 [internal function]: PostController->actionView('14')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=14
HTTP_REFERER=http://localhost/blog/index.php?r=post/create
---
2024/09/23 15:57:22 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:161
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('15')
#1 [internal function]: PostController->actionView('15')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=15
HTTP_REFERER=http://localhost/blog/index.php?r=post/create
---
2024/09/23 15:57:32 [error] [exception.CHttpException.404] CHttpException: The requested page does not exist. in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:161
Stack trace:
#0 C:\Apache24\htdocs\blog\protected\controllers\PostController.php(51): PostController->loadModel('16')
#1 [internal function]: PostController->actionView('16')
#2 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#3 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#4 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/view')
#14 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=post/view&id=16
HTTP_REFERER=http://localhost/blog/index.php?r=post/create
---
2024/09/26 18:19:45 [error] [exception.CHttpException.403] CHttpException: Ошибка доступа. Не твоё - не трогай! in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:99
Stack trace:
#0 [internal function]: PostController->actionUpdate('4')
#1 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#2 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#3 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#7 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\yii\framework\web\CWebApplication.php(282): CController->run('update')
#12 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/update')
#13 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/blog/index.php?r=post/update&id=4
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=4
---
2024/09/26 18:21:48 [error] [exception.CHttpException.403] CHttpException: Ошибка доступа. Не твоё - не трогай! in C:\Apache24\htdocs\blog\protected\controllers\PostController.php:115
Stack trace:
#0 [internal function]: PostController->actionDelete('4')
#1 C:\yii\framework\web\actions\CAction.php(114): ReflectionMethod->invokeArgs(Object(PostController), Array)
#2 C:\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#3 C:\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#7 C:\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 C:\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#12 C:\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/delete')
#13 C:\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\Apache24\htdocs\blog\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/blog/index.php?r=post/delete&id=4
HTTP_REFERER=http://localhost/blog/index.php?r=post/view&id=4
---
